Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -740,7 +745,8 @@ UpdateFspConfig (
     }
 
     // TSN feature support
-    if (IsPchS () || IsPchN () ) {
+    if (IsPchS () || IsPchN () ||
+      (IsPchP() && 0 == AsciiStrCmp(GetCpuName(),"RaptorLake"))) {
```

Comment:
I think these conditions could also be removed since the configuration data should be able to handle it.

Reply:
I agree. I'll make the change.